cmd/test/test.go

```diff
@@ -0,0 +1,211 @@
+package main
```

Comment:
The fact that you have this and I have my version of this, I ok to commit this to the code base as it makes development easier. Can we have some header comments what this script is intended to do? and rename `test` to something like `pe-test`?